lib/node-writable.js

```diff
@@ -0,0 +1,21 @@
+var Writable = require('stream').Writable,
```

Comment:
Once upon a time we did (back in 1.x according to git history). I think we dropped it because one of our `devDependencies` is no longer compatible with the version of `npm` that comes with 0.8 on travis.

If you're concerned we could make `asNodeWritable` return `this` if `require('stream').Writable` doesn't exist. Thoughts?